File: Backend/src/Repos/Models/XML_Model/XML7.js
const mongoose = require('mongoose');

const XML7Schema = new mongoose.Schema({
  id:                    { type: Number, required: false },
  Ma_Lk:                 { type: String, default: null },
  So_Luu_Tru:            { type: String, default: null },
  Ma_Yte:                { type: String, default: null },
  Ma_Khoa_Rv:            { type: String, default: null },
  Ngay_Vao:              { type: String, default: null },
  Ngay_Ra:               { type: String, default: null },
  Ma_Dinh_Chi_Thai:      { type: Number, default: null },
  NguyenNhan_DinhChi:    { type: String, default: null },
  ThoiGian_DinhChi:      { type: String, default: null },
  Tuoi_Thai:             { type: String, default: null },
  Chan_DoAn_Rv:          { type: String, default: null },
  Pp_DieuTri:            { type: String, default: null },
  Ghi_Chu:               { type: String, default: null },
  Ma_Ttdv:               { type: String, default: null },
  Ma_Bs:                 { type: String, default: null },
  Ten_Bs:                { type: String, default: null },
  Ngay_Ct:               { type: String, default: null },
  Ma_Cha:                { type: String, default: null },
  Ma_Me:                 { type: String, default: null },
  Ma_The_Tam:            { type: String, default: null },
  Ho_Ten_Cha:            { type: String, default: null },
  Ho_Ten_Me:             { type: String, default: null },
  So_Ngay_Nghi:          { type: Number, default: null },
  NgoaiTru_TuNgay:       { type: String, default: null },
  NgoaiTru_DenNgay:      { type: String, default: null },
  Du_Phong:              { type: String, default: null }
}, {
  timestamps: true,
  strict: false
});

module.exports = mongoose.model('XML7', XML7Schema);

File: WPF_GiamDinhBaoHiemYTe/Repos/Model/Patietn_XML_data/XML7.cs
using CommunityToolkit.Mvvm.ComponentModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPF_GiamDinhBaoHiem.Repos.Model
{
    public partial class XML7 : ObservableObject
    {
        [ObservableProperty] private int id;
        [ObservableProperty] private string? ma_Lk;
        [ObservableProperty] private string? so_Luu_Tru;
        [ObservableProperty] private string? ma_Yte;
        [ObservableProperty] private string? ma_Khoa_Rv;
        [ObservableProperty] private string? ngay_Vao;
        [ObservableProperty] private string? ngay_Ra;
        [ObservableProperty] private int? ma_Dinh_Chi_Thai;
        [ObservableProperty] private string? nguyenNhan_DinhChi;
        [ObservableProperty] private string? thoiGian_DinhChi;
        [ObservableProperty] private string? tuoi_Thai;
        [ObservableProperty] private string? chan_DoAn_Rv;
        [ObservableProperty] private string? pp_DieuTri;
        [ObservableProperty] private string? ghi_Chu;
        [ObservableProperty] private string? ma_Ttdv;
        [ObservableProperty] private string? ma_Bs;
        [ObservableProperty] private string? ten_Bs;
        [ObservableProperty] private string? ngay_Ct;
        [ObservableProperty] private string? ma_Cha;
        [ObservableProperty] private string? ma_Me;
        [ObservableProperty] private string? ma_The_Tam;
        [ObservableProperty] private string? ho_Ten_Cha;
        [ObservableProperty] private string? ho_Ten_Me;
        [ObservableProperty] private int? so_Ngay_Nghi;
        [ObservableProperty] private string? ngoaiTru_TuNgay;
        [ObservableProperty] private string? ngoaiTru_DenNgay;
        [ObservableProperty] private string? du_Phong;
    }
}
